ymer-test-junit4/src/test/java/com/avanza/ymer/YmerConverterTestBaseTest.java

```diff
@@ -359,7 +311,7 @@ private static void assertPasses(ThrowingRunnable testRun) {
 		try {
 			testRun.run();
 		} catch (AssertionError e) {
-			fail("Expected test to pass");
+			fail("Expected test to pass, but failed with \"" + e.getMessage() + "\"");
```

Comment:
Maybe throw a new exception here to get the stack trace info as well?

Reply:
I simplified the method to only catch `AssertionError`